### PR TITLE
lxgw-fusionkai: init at 24.134

### PR DESCRIPTION
**Pull request by hellodword**
## Description of changes

A Klee One based Chinese font.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/lx/lxgw-fusionkai/package.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    repo = "FusionKai";
+    rev = version;
+    hash = "sha256-pEISoFEsv8SJOGa2ud/nV1yvl8T9kakfKENu3mfYA5A=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/truetype
+    mv *.ttf $out/share/fonts/truetype
```
Comment (NickCao) [inline pkgs/by-name/lx/lxgw-fusionkai/package.nix:22]: We can use the `install` command, as in `pkgs/data/fonts/cascadia-code/default.nix`

File: pkgs/by-name/lx/lxgw-fusionkai/package.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lxgw-fusionkai";
+  version = "v24.134";
```
Comment (NickCao) [inline pkgs/by-name/lx/lxgw-fusionkai/package.nix:9]: `version` should not contain the `v` prefix.

File: pkgs/by-name/lx/lxgw-fusionkai/package.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    mkdir -p $out/share/fonts/truetype
+    mv *.ttf $out/share/fonts/truetype
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/lxgw/FusionKai";
+    description = "「缝合楷」：将「霞鹜文楷 GB」「芫荽」「Klee One」等字体相互合并制成的以各地字形为主的 Klee One 衍生字体。 ";
```
Comment (NickCao) [inline pkgs/by-name/lx/lxgw-fusionkai/package.nix:29]: I don't remember seeing other instances of non-latin characters in description.

File: pkgs/by-name/lx/lxgw-fusionkai/package.nix [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    license = licenses.ofl;
+    platforms = platforms.all;
+  };
```
Comment (NickCao) [inline pkgs/by-name/lx/lxgw-fusionkai/package.nix:32]: `maintainers` must be set for new packages.

File: pkgs/by-name/lx/lxgw-fusionkai/package.nix (hunk 1 above)
Comment (lebensterben) [reply pkgs/by-name/lx/lxgw-fusionkai/package.nix:22]: Is this a documented requirement to prefer to use "install"?

File: maintainers/maintainer-list.nix [hunk 5]
```diff
@@ -8200,6 +8200,11 @@
     githubId = 2405974;
     name = "Sébastian Méric de Bellefon";
   };
+  hellodword = {
+    github = "hellodword";
+    githubId = 46193371;
+    name = "hellodword";
+  };
```
Comment (NickCao) [inline maintainers/maintainer-list.nix:8207]: Please split the maintainer addition to its own commit, `maintainers: add hellodword`

File: pkgs/by-name/lx/lxgw-fusionkai/package.nix [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    runHook preInstall
+
+    install -Dm644 *.ttf -t $out/share/fonts/truetype
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/lxgw/FusionKai";
+    description = "A variant of the Klee One font, formed by merging LxgwWenkaiGB iansui Klee and others, emphasizing the glyphs from different regions.";
```
Comment (NickCao) [inline pkgs/by-name/lx/lxgw-fusionkai/package.nix:28]: `description` should not start with the definite or an indefinite article (the "A"), and should not end with a period, and also be short, just one sentence. So what about `Simplified Chinese font derived from LXGW WenKai GB, iansui and Klee One`

**Review by lebensterben — APPROVED** [review]

**Review by NickCao — APPROVED** [review]
